### DISCUSSION TITLE
Change to a specific skins/clothes

### DISCUSSION THREAD

**Discussion by Xenia-class** [Luke100000/minecraft-comes-alive#663] [Q&A]
I have a question but first I'm going to write the situation:

 If an npc appears with an exact clothing (for example, a special or custom clothing) and that npc doesn't have a job, but gets one (because of that the clothes change) it would be possible for them to change to a specific clothe and not to a random assigned for that job?

Or do I have to change them manually? Because after testing a little i saw that they usually change clothes sometimes 

Maybe a way to block certain clothes so they'll not change

**Comment by Luke100000** [comment]
They only reset on profession change. Trade once to lock the profession.

**Reply by Xenia-class** [reply]
Ah I see, thank you!
